File: C++/max sum.cpp
#include <iostream>

using namespace std;

int main() {
	int n, i, s1 = 0, s2 = 0;
	cin >> n;
	int nums[n];
	for (i = 0; i<n; i++) {
		cin >> nums[i];
	}
	for (i = 0; i < n; i+=2) {
		s1 += nums[i];
	}
	for (i = 1; i < n; i+=2) {
		s2 += nums[i];
	}
	if (s1 > s2) {
		cout << s1 << endl;
	}
	else {
		cout << s2 << endl;
	}

	return 0;
}

File: Python/max sum.py
n = int(input())
nums = []
s1=0
s2=0
for i in range(n):
	nums.append(int(input()))
for i in range(0, n, 2):
	s1 += nums[i]
for i in range(1, n, 2):
	s2 += nums[i]

print(max(s1, s2))